rds/rds_provider.go

```diff
@@ -116,7 +125,6 @@ func (r *RDS) ensureSubnets(db *crd.Database) (string, error) {
 			DBSubnetGroupDescription: aws.String(subnetDescription),
 			DBSubnetGroupName:        aws.String(subnetName),
 			SubnetIds:                r.Subnets,
-			Tags:                     []rds.Tag{{Key: aws.String("DBName"), Value: aws.String(db.Spec.DBName)}},
```

Comment:
Still think a tag might be in order, to indicate it's managed by k8s-rds?

Reply:
I'll add it.

Reply:
I added the tag "Warning" just like terraform does.